.buildkite/pipeline.yml

```
@@ -60,6 +60,20 @@ steps:
       - github_commit_status:
           context: E2E Tests
 
+  - label: ":chart_with_upwards_trend: Performance Metrics"
+    key: metrics
+    agents:
+      queue: mac
+    command: bash .buildkite/commands/run-metrics-tests.sh
```

Nitpick: I think calling `bash` is redundant here because the script already has `!#/usr/bin/env bash`. Most of the `.sh` scripts we use as the `command` steps in other projects are invoked that way.


```suggestion
    command: .buildkite/commands/run-metrics-tests.sh
```

---

Thanks for confirming that.